Assets/MixedRealityToolkit/Inspectors/MixedRealityToolkitInspector.cs

```diff
@@ -118,11 +143,267 @@ public override void OnInspectorGUI()
 
             if (activeProfile.objectReferenceValue != null)
             {
-                UnityEditor.Editor activeProfileEditor = CreateEditor(activeProfile.objectReferenceValue);
-                activeProfileEditor.OnInspectorGUI();
+                int searchDisplayToolbarIndex = SessionState.GetInt(searchDisplayToolbarIndexKey, 0);
+                searchDisplayToolbarIndex = GUILayout.Toolbar(searchDisplayToolbarIndex, searchDisplayToolbarButtons);
+                SessionState.SetInt(searchDisplayToolbarIndexKey, searchDisplayToolbarIndex);
+                switch (searchDisplayToolbarIndex)
+                {
+                    case 0:
+                    default:
+                        // For configure, show the default inspector GUI
+                        UnityEditor.Editor activeProfileEditor = CreateEditor(activeProfile.objectReferenceValue);
+                        activeProfileEditor.OnInspectorGUI();
+                        break;
+
+                    case 1:
+                        // For search, draw our custom search GUI
+                        DrawSearchGUI(activeProfile.objectReferenceValue);
+                        break;
+                }
+            }
+        }
+
+        #region search
+
+        private void DrawSearchGUI(UnityEngine.Object activeProfileObject)
+        {
+            EditorGUILayout.Space();
+
+            SearchConfig config = new SearchConfig()
```

Comment:
This whole session state and population tracking might honestly be worth it's own helper class

Reply:
SearchConfigPreferences or SearchConfigInspector or some other better name/utility because I am terrible at naming

Reply:
Good idea.